handlers/geospatialdata.go

```diff
@@ -19,7 +19,7 @@ import (
 // @Success 200 {object} interface{}
 // @Failure 500 {object} SimpleResponse
 // @Router /geospatialdata [get]
-func GeospatialData(fs *filestore.FileStore) echo.HandlerFunc {
+func GeospatialData(fs *filestore.FileStore, destinationCRS int) echo.HandlerFunc {
```

Comment:
Recommend making the only input to this function the appConfig struct, and pulling the filestore and crs from that. It will make your handler function cleaner in main.go

Reply:
Good point, done!